ocd-1903.sql

```diff
@@ -0,0 +1,38 @@
+UPDATE openchpl.global_user_permission_map as gupm
+SET user_permission_id_user_permission = 2
+WHERE user_id = 22;
+
```

Comment:
So if someone created an ACB_STAFF or ATL_STAFF user between now and the deployment of this (which I realize is unlikely), those new users wouldn't be included in this update. Would it be better to "SET user_permission_id_user_permission = 2 WHERE user_permission_id_user_permission = 3" and a second similar statement for ATL_STAFF?